### PR TITLE
Theme : add links to a donation page and the about page.

### LINKED ISSUE #931: UI question: add help & donate links in the top right corner.
The oscar theme provides some easy to see links to the help.
searx.be adds a link to a donation page.

How to add these links ?

A suggestion here:

## On desktop

image

image

### Alternative for the result page
image

## On tablet

image

### Alternative for the result page
image

### index page

| with labels| only icons|
| --- | --- |
| image | image |

### result page
The result page remains the same
image

---

icons ( https://ionic.io/ionicons ):
* help-circle-outline : there is an issue with the SVG, two attributes are missing `stroke="currentColor" fill="currentColor"` on the `circle` element. They can be added by CSS (?) or we can provide our own version.
* heart-outline

### PR DESCRIPTION
**Pull request by dalf**
### front page

image

image

image

image

### results

image

image

image

image

### preferences

image

### donate page

image

## Why is this change important?

* Add the missing "About" button
* Add a new "Donate" button.

## Author's checklist

`help-circle-outline.svg` is copied into the git repository because the ion icon version is broken: the circle is missing these attributes `stroke="currentColor" stroke-width="16" fill="currentColor"`.

## Related issues

Close https://github.com/searxng/searxng/issues/931

### CODE REVIEW

**Comment by unixfox** [comment]
There is no existing default donate page, is that expected?

image

**Comment by dalf** [comment]
> There is no existing default donate page, is that expected?

Not intended, fixed.

**Review by unixfox — COMMENTED** [review]
I think the links should be clickable.

File: searx/infopage/en/donate.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Donate to SearXNG
+
+* Credit/debit card and bank transfer
+    * [Liberapay] (recurrent donation)
+    * [Buy Me a Coffee] (one time donation)
+
+* Cryptocurrency
+    * Bitcoin: bc1qn3rw8t86h05cs3grx2kmwmptw9k4kt4hyzktqj (Segwit compatible)
+    * Bitcoin cash: bitcoincash:qpead2yu482e3h9amy5zk45l8qrfhk59jcpw3cth9e
```
Comment (unixfox) [inline searx/infopage/en/donate.md]: Why is there a `bitcoincash:` before the address? With the new bitcoin cash address, you can't unintentionally send bitcoin to a bitcoin cash format address.

**Review by unixfox — APPROVED** [review]

**Review by mrpaulblack — APPROVED** [review]
LGTM :+1: 

Thanks a lot the impl. is great and this is a long awaited feature for the simple theme :tada:

**Pull request merged**